packages/react-router/lib/types/route-module.ts

```diff
@@ -260,3 +260,13 @@ type __tests = [
     >
   >
 ];
+
+/**
+ * A brand that can be applied to a type to indicate that it will serialize
+ * to a specific type when transported to the client from a loader.
+ * Only use this if you have additional serialization/deserialization logic
+ * in your application.
+ */
+export type SerializesTo<T> = {
+  $__RR_SerializesTo?: [T];
```

Comment:
Is the `$__` prefix a widely used convention? I was thinking of `__ReactRouter_` prefix instead of `$__RR_` just for increased clarity, but wasn't sure if there were conventions I'm unaware of

Reply:
I'm perfectly fine with `__ReactRouter_SerializesTo`, too - I just hadn't seen any precedence and wanted to be extra sure :)

We use the `$ ` (even with a space) in GraphQL because it's not allowed per spec and we avoid collisions, but here it's not really necessary.

I'll fix this up to be `__ReactRouter_SerializesTo` later!

Reply:
Okay, made that change!